db/migrate/20170614174723_add_user_id_to_wishlists.rb

```diff
@@ -0,0 +1,5 @@
+class AddUserIdToWishlists < ActiveRecord::Migration[5.0]
+  def change
+    add_column :wishlists, :user_id, :integer
```

Comment:
Consider adding a foreign key for this as well as indexing - https://robots.thoughtbot.com/referential-integrity-with-foreign-keys